### PR TITLE
feat: Add UI support for VPN list/create/delete

### PR DESCRIPTION
**Pull request by floreks**
## Summary

- Added VPN client list to the account view
- Added VPN client list to my profile view

Ref: [Figma](https://www.figma.com/file/xj6gU4Jk6qR9gtfb3Bc9jm/VPN-in-Console?node-id=1%3A24467&t=MsLnSH71xzU5u1Cg-0)
~Blocked by: https://github.com/pluralsh/design-system/pull/424~

## Test Plan

Locally

## Checklist

- [ ] If required, I have updated the Plural documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added a meaningful title and summary to convey the impact of this PR to a user.
- [ ] I have added relevant labels to this PR to help with categorization for release notes.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203980256742771

### CODE REVIEW

File: assets/package.json [hunk 1]
```diff
@@ -147,5 +147,8 @@
     "vite-tsconfig-paths": "4.0.5",
     "vitest": "0.28.4",
     "wait-on": "6.0.1"
+  },
+  "resolutions": {
```
Comment (floreks) [inline assets/package.json:151]: Remember to remove that :slightly_smiling_face:

File: assets/src/components/vpn/graphql/mutations.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { gql } from 'apollo-boost'
+
+const CreateWireguardPeer = gql`
+  mutation CreateWireguardPeer($email: String, $name: String!, $userId: ID) {
+    createPeer(email: $email, name: $name, userId: $userId) {
+      raw
```
Comment (michaeljguarino) [inline assets/src/components/vpn/graphql/mutations.ts:6]: I think you can query the status and other fields here now w/o it erroring

**Review by michaeljguarino — APPROVED** [review]

File: assets/src/components/account/vpn/VPNHeaderActions.tsx [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+    portrait: <AppIcon
+      key={user?.id}
+      name={user?.name}
+      url={user?.profile ?? ''}
+      spacing={user?.profile ? 'none' : undefined}
+      size="xxsmall"
+    />,
+  })), [users])
+
+  useEffect(() => onFilter(selectedKeys), [selectedKeys])
```
Comment (dogmar) [inline assets/src/components/account/vpn/VPNHeaderActions.tsx:84]: Missing effect dependency 'onFilter'

File: assets/src/components/cluster/pods/Pods.tsx [hunk 4]
```diff
@@ -111,8 +111,8 @@ export default function AllPods() {
       if (!data?.podDelta) return prev
       const { podDelta: { delta, payload } } = data
 
-      if (delta === 'CREATE') return { ...prev, cachedPods: uniqBy([payload, ...prev.cachedPods], p => `${p.metadata.name}:${p.metadata.namespace}`) }
-      if (delta === 'DELETE') return { ...prev, cachedPods: prev.cachedPods.filter(p => !isEqual(p, payload)) }
+      if (delta === 'CREATE') return { ...prev, cachedPods: uniqBy([payload, ...prev.cachedPods ?? []], p => `${p.metadata.name}:${p.metadata.namespace}`) }
+      if (delta === 'DELETE') return { ...prev, cachedPods: prev.cachedPods?.filter(p => !isEqual(p!, payload)) }
```
Comment (dogmar) [inline assets/src/components/cluster/pods/Pods.tsx:115]: Oh thanks! I was just about to go through and fix all these TS errors, and now I see you've taken care of it. :)

**Review by dogmar — APPROVED** [review]

File: assets/src/components/vpn/graphql/mutations.ts (hunk 2 above)
Comment (floreks) [reply assets/src/components/vpn/graphql/mutations.ts:6]: Yeah, but I am not using the result of create mutation in any way right now.

File: assets/src/components/cluster/pods/Pods.tsx (hunk 4 above)
Comment (floreks) [reply assets/src/components/cluster/pods/Pods.tsx:115]: Ye, I also added `tsc` call to the `build` script.

**Pull request merged**